src/MIDebugEngine/Engine.Impl/DebuggedProcess.cs

```diff
@@ -858,6 +876,40 @@ private void Dispose()
             }
         }
 
+        private async Task VerifySourceFileTimestamp(ulong addr, string sourceFilePath)
+        {
+            await this.EnsureModulesLoaded();
+
+            string targetModulePath = this._launchOptions.ExePath;
+            DebuggedModule targetModule = this._moduleList.FirstOrDefault(m => m.AddressInModule(addr));
+            if (targetModule != null)
+            {
+                targetModulePath = targetModule.Name;
+            }
+
+            if (!File.Exists(sourceFilePath) || !File.Exists(targetModulePath))
```

Comment:
File.Exists will return false if you pass null / empty string, so this is handled.